packages/server/graphql/types/Organization.ts

```diff
@@ -20,7 +22,7 @@ import Team from './Team'
 import TierEnum from './TierEnum'
 import User from './User'
 
-const Organization = new GraphQLObjectType<any, GQLContext>({
+const Organization: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<any, GQLContext>({
```

Comment:
interesting that TS has to infer `any` because of the circular reference in the fields!